### PR TITLE
refactor(Travis): Replace mono_repo with hand-written script.

### PR DESCRIPTION
**Pull request by matanlurey**
Closes https://github.com/dart-lang/angular/pull/981.

### CODE REVIEW

**Comment by kevmoo** [comment]
Do we feel mono_repo won't "scale" to this setup?

**Comment by kevmoo** [comment]
> Do we feel mono_repo won't "scale" to this setup?

Saw the internal discussion. Deleting.

File: dev/travis/postfix.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Create a merged output build/.
+before_deploy:
+  - pushd examples/hacker_news_pwa
```
Comment (kevmoo) [inline dev/travis/postfix.yaml]: We've already built in a stage before, right? Why do it again?

...or just running it expecting a `no-opt`?

Comment (matanlurey) [reply dev/travis/postfix.yaml]: Right, expectation is a no-op, but I'm not sure deployment can see the build cache, so being extra cautious.

Comment (kevmoo) [reply dev/travis/postfix.yaml]: Well. It's weird.

`the -d example/hacker_new...` condition makes sure that directory exists.

So the deploy (and pre deploy) will only run in that case.

...I guess it should be a no-op, but would be good to validate.

Comment (matanlurey) [reply dev/travis/postfix.yaml]: Definitely. Let's watch the logs and see what happens.

Comment (kevmoo) [reply dev/travis/postfix.yaml]: Won't see anything here. Only when it lands in master.

Unless you want to push this to a branch here and enable it – to sanity check...

File: .travis.yml [hunk 2]
```diff
@@ -1,174 +1,261 @@
...
+
+    - stage: presubmit
+      script: ./tool/travis.sh analyze
       env: PKG="examples/hacker_news_pwa"
-      dart: dev
-    - stage: build
-      script: ./tool/travis.sh command_7
+
+    - stage: building
+      script: ./tool/travis.sh build
```
Comment (kevmoo) [inline .travis.yml]: should be `build:release`?

File: .travis.yml [hunk 3]
```diff
@@ -1,174 +1,261 @@
...
+          - examples/hello_world_no_reflector/.dart_tool
 
-stages:
-  - analyze
-  - build
-  - unit_test
+# Create a merged output build/.
+before_deploy:
+  - pushd examples/hacker_news_pwa
+  - pub run build_runner build -o build --fail-on-severe
```
Comment (kevmoo) [inline .travis.yml]: should have `--config release` ?

File: .travis.yml (hunk 2 above)
Comment (kevmoo) [reply .travis.yml]: @matanlurey ?

Also, the deploy command won't run if the `build/web` directory isn't created.

Do we need `build:release:output`?

Comment (matanlurey) [reply .travis.yml]: Why? We use a `before_deploy` step.

**Pull request closed without merging**